util/queue.py

```diff
@@ -66,7 +66,7 @@ class Queue(multiprocessing.queues.Queue):
     """
 
     def __init__(self, *args, **kwargs):
-        super(Queue, self).__init__(*args, **kwargs)
+        super(Queue, self).__init__(*args, **kwargs, ctx=multiprocessing.get_context())
```

Comment:
Please list this before **kwargs, so that it also work with Python 2 (see the [failing Travis CI test](https://travis-ci.org/vterron/lemon/builds/612538946)).